mautic/mautic
# Empty theme form threw an error - fixed

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | /
| Deprecations? | /

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Submitting a theme upload form resulted in error instead of a validation message.

#### Steps to test this PR:
1. Apply this PR.
2. Repeat the test - should see a validation error message.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to the Theme manager
2. Do not select any package and hit Submit - you should see a PHP error.
